projects/ngrx.io/content/guide/store/action-groups.md

```diff
@@ -0,0 +1,67 @@
+# Action Groups
+
+Action groups is a feature that allows you to group actions together. While writing actions, the actions in most of the cases looks like below.
+
+<code-example header="products-page.actions.ts">
+import { createAction, props } from '@ngrx/store';
+// defining an action without payload
+export const opened = createAction('[Products Page] Opened');
+// defining an action with payload using the `props` function
+export const paginationChanged = createAction(
+  '[Products Page] Pagination Changed',
+  props&lt;{ page: number; offset: number }&gt;()
+);
+// defining an action with payload using the props factory
+export const queryChanged = createAction(
+  '[Products Page] Query Changed',
+  (query: string) => ({ query })
+);
+</code-example>
+
+
+With the help of this new `createActionGroup` api, we can avoid the above code and write the following code.
+
+<code-example header="products-page.actions.ts">
+
+import { createActionGroup, emptyProps, props } from '@ngrx/store';
+export const ProductsPageActions = createActionGroup({
+  source: 'Products Page',
+  events: {
+    // defining an event without payload using the `emptyProps` function
+    'Opened': emptyProps(),
+    
+    // defining an event with payload using the `props` function
+    'Pagination Changed': props&lt;{ page: number; offset: number }&gt;(),
+    
+    // defining an event with payload using the props factory
+    'Query Changed': (query: string) => ({ query }),
+  }
+});
+</code-example>
+
+After moving to the new `createActionGroup` api, we can use the following code to import and dispatch the action.
```

Comment:
```suggestion
To dispatch an action from the group, import the group and invoke an action.
This returns an action that you can then dispatch to the store.
```

This snippet should also show how to dispatch an action to the store.
The current snippet only invokes the action, but doesn't dispatch it.